src/patternfly/components/Expandable/docs/code.md

```diff
@@ -0,0 +1,21 @@
+## Accessibility
+
+| Attribute | Applied To | Outcome |
+| -- | -- | -- |
+| `aria-expanded="true"` | `.pf-c-expandable__toggle` | Indicated that the expandable content is visible. **Required** |
+| `hidden` | `.pf-c-expandable__content` | Indicates that the expandable content is hidden. **Required**. |
+| `aria-hidden="true"` | `.pf-c-expandable__toggle-icon` | Hides from screen readers. |
+| `aria-controls="[id of element controlled]"` | `.pf-c-expandable__toggle-icon` | Identifies the section controlled by the toggle button. |
+
+
+## Usage
+
+| Class | Applied To | Outcome |
+| -- | -- | -- |
+| `.pf-c-expandable` | `<div>` | Initiates the expandable component. **Required** |
+| `.pf-c-expandable__section` | `<div>` | Initiates the expandable component section. **Required** |
```

Comment:
I don't think `__section` needs to be a part of this component... this component should just handle the expandable toggle/content, and that should be able to be plugged into whatever. The content above it should just be whatever, probably form data, and the footer below, if it were buttons, would be the buttons in the form, not part of this component. So you can remove `__footer`, too

Reply:
Okay makes sense